whatis277/harvest
# pear: Fix dockerfile nginx support

In https://github.com/whatis277/harvest/pull/72, the integration was bad

Changes to `index.html` didn't get reflected
Fixes that by creating a volume matching `pear/html` with `nginx/html`

Also moves `index.html` into its own directory
This is mainly to only copy specific files

Testing instructions:
1. `dc up --build pear`
2. Visit [`http://localhost:8081`](http://localhost:8081)
3. Modify something in `index.html`
4. Refresh the page
5. Ensure the changes are reflected
